setup.py

```diff
@@ -41,7 +41,6 @@
         'pyproj>=2.1'
         'pyyaml',
         'scitools-iris>=2.2',
-        'scikit-learn',
         'seawater',
```

Comment:
Please, do not remove package from setup.py. If it is already installed through conda, it will not be overwritten but if not it can lead to lots of issues

Reply:
huh? no, if it's there it's installed via `pip`

Reply:
and I want it through conda

Reply:
`pip` will not install anything that already has a compatible version installed, so if it's already installed by conda, pip will skip installation.

Reply:
that is correct, I got st00pidly confused, my bad :beer: